### PR TITLE
[stable/kibana]: allow adding volumes & mounts

### PR DESCRIPTION
**Pull request by unclejack**
#### What this PR does / why we need it:
This PR makes it possible to add volumes and VolumeMounts to the kibana pod.

#### Special notes for your reviewer:
Please let me know if I should make any other changes.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by compleatang** [comment]
@unclejack just ping me whenever you have had a chance to fixup the linter errors.

**Comment by unclejack** [comment]
@compleatang: Thank you for the heads up. I've made one more change to make it possible to disable the internal port. This is needed when a proxy sidecar is used. Please take another look.

**Comment by compleatang** [comment]
@unclejack this looks reasonable to me. can you also add a test for the feature under the /ci directory of the chart so that future modifications don't break things -- especially the sidecar portion of hte PR. there are other examples in there as well.

**Comment by unclejack** [comment]
@compleatang: I've added a new YAML to try to test the disabled internal service port. Please let me know if I should add some more tests for all the other changes. I couldn't find a single chart which had a CI test for extraContainers, extraVolumes and extraVolumeMounts.

**Comment by yurrriq** [comment]
The `disableInternalPort` logic opposite to my intuition. I would expect the port to be disabled when `disableInternalPort=true`, instead of when it's `false`. Is this a typo?

File: stable/kibana/templates/deployment.yaml [hunk 1]
```diff
@@ -106,10 +106,12 @@ spec:
         - name: "{{ $key }}"
           value: "{{ $value }}"
         {{- end }}
+{{- if .Values.service.disableInternalPort }}
```
Comment (yurrriq) [inline stable/kibana/templates/deployment.yaml:109]: As per [my comment](https://github.com/helm/charts/pull/10763#issuecomment-457741836), it seems like the intended behaviour would warrant the following instead:
```yaml
{{- if not .Values.service.disableInternalPort }}
```

Comment (unclejack) [reply stable/kibana/templates/deployment.yaml:109]: @yurrriq: I'll send another PR to fix this.

Comment (unclejack) [reply stable/kibana/templates/deployment.yaml:109]: This is fixed in https://github.com/helm/charts/pull/10932

**Pull request merged**